docs/content/API-Reference/SQL-API.mdx

```diff
@@ -369,6 +369,29 @@ module.exports = {
 };
 ```
 
+## Security Context (Row-Level Security)
+
+Cube.js provides security context functionality, which can be used in the
+[Dynamic Schema Creation][ref-dynamic-schemas] or [`queryRewrite`][ref-config-queryrewrite] property in your [`cube.js`
+configuration file][ref-config-js].
+
+By default, SQL API uses current user's security context, but there is way how to change it.
```

Comment:
```suggestion
By default, the SQL API uses the current user's Security Context, but this behaviour can be modified so that certain users are allowed to switch. To do this, we must first define which user is allowed to change Security Context:
```